includes/transformer/class-post.php

```diff
@@ -57,6 +57,7 @@ public function to_object() {
 			$object->set_sensitive( true );
 			$object->set_summary( $content_warning );
 			$object->set_summary_map( null );
+			$object->set_dcterms( array( 'subject' => $content_warning ) );
```

Comment:
Optional: I think it would be cool if there was a unit test that covered this showing up in a post with content warning. It doesn't look like the ones in `Test_Outbox` cover that scenario